### DISCUSSION TITLE
Discussion: TextMate bundles translator

### DISCUSSION THREAD

**Discussion by ghost** [SpartanJ/ecode#267] [General]
Is writing a tool to translate TextMate bundles to native `ecode` syntax highlighter format or just supporting TextMate bundles directly (#191) easier?

**Comment by SpartanJ** [comment]
Please open discussion in... [Discussions](https://github.com/SpartanJ/ecode/discussions).

**Reply by SpartanJ** [reply]
It should be slightly easier once ecode supports regexs in grammars.
